src/main/webapp/pages/password/password_reset.html

```diff
@@ -1,37 +1,34 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org"
-      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{template/base}">
-<head>
-    <title th:text="${user.label}">_USER PAGE_</title>
-</head>
-<body>
-<main layout:fragment="content" class="container main-area" role="main" property="mainContentOfPage">
-    <h1 th:text="#{password.reset.title(${user.label})}">_Reset User Password_</h1>
-    <div class="col-md-6">
-        <form role="form" action="/password_reset/1"
-              th:action="@{/password_reset/{id}(id=${passwordReset.id})}" method="post">
-            <div class="alert alert-danger"
-                 th:if="${expired}"
-                 th:text="#{password.reset.password_expired}">
-                _Your password has expired.  Please enter a new password._
-            </div>
-            <div th:replace="templates/password :: passwordInputs (hidden=false, required=true)"></div>
-            <button type="submit"
-                    class="btn btn-primary submit js-submit-btn t-submit-btn"
-                    th:text="#{form.password-set}">
-                Set Password
-            </button>
-            <a class="btn btn-default" href="#" th:href="@{/}" th:text="#{form.btn.cancel}">Cancel</a>
-        </form>
+<html xmlns:th="http://www.thymeleaf.org" th:lang="${#locale}">
+  <head>
+    <title>Reset Password</title>
```

Comment:
I18N

Reply:
Added in 66fd23c47ca7b634ae1918ac77341a2fdf53048e